ubuntudesign/www.ubuntu.com
# Ensure resources are loaded through HTTPS

- Make some ambiguous resources explicitly load over HTTPS
- Fix styling for sliders
- For domains that we *know* support HTTPS (like insights.ubuntu.com), make *all links* use HTTPS

QA
--

Click around the site, especially on touched pages, and make sure everything works and looks right.
